app/_layouts/plugin-ee.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+sitemap: true
+id: page-plugin
+header_btn_text: Report Bug
+header_btn_href: mailto:support@mashape.com?subject={{ page.header_title }} Plugin Bug
```

Comment:
Are we still support@mashape?

Reply:
This needs to be updated and I am doing so locally already as I am merging #622